Linux build: The dependency target "inject_debug_resources_coreclr" of target "runtimeinfo" does not exist.

Sometime recently, I've become unable to build dotnet/runtime on Linux Ubuntu 20.04.2 LTS.

Steps:

1. `git pull` (HEAD is now b80ddefa8db351f4ed697a686d8b6ba1dd927c52)
2. `git clean -xdf` (just for good measure)
3. `./build.sh -s clr+libs -rc release`

Result:

<pre>
  -- Configuring done
  CMake Error at debug/runtimeinfo/CMakeLists.txt:34 (add_dependencies):
    The dependency target "inject_debug_resources_coreclr" of target
    "runtimeinfo" does not exist.
  
  
  -- Generating done
  CMake Generate step failed.  Build files cannot be regenerated correctly.
  /code/personal/dotnet/runtime/src/coreclr
</pre>

I don't think anything in particular changed in my environment, and nor do I see any other error higher up in the build logs. I tried a branch that is a few weeks old and was able to build it successfully.

Any assistance getting the build working again would be appreciated.

--- a/src/coreclr/dlls/mscoree/coreclr/CMakeLists.txt
+++ b/src/coreclr/dlls/mscoree/coreclr/CMakeLists.txt
@@ -243,6 +243,9 @@ if(CLR_CMAKE_TARGET_WIN32)
         add_custom_target(inject_debug_resources_singlefilehost ALL DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR}/inject_debug_resources_singlefilehost.timestamp)
         add_dependencies(runtime inject_debug_resources_singlefilehost)
     endif()
+else()
+    add_custom_target(inject_debug_resources_coreclr DEPENDS coreclr)
+    add_custom_target(inject_debug_resources_singlefilehost DEPENDS singlefilehost)
 endif(CLR_CMAKE_TARGET_WIN32)
 
 # add the install targets

